platform/state/platform_state.proto

```diff
@@ -195,3 +211,134 @@ message MinimumJudgeInfo {
      */
     uint64 minimum_judge_ancient_threshold = 2;
 }
+
+/**
+ * A network address book.<br/>
+ * The address book records the address of every known consensus node that
+ * participates in the network, including `0 weight` nodes.<br/>
+ */
+message AddressBook {
+  /**
+   * A consensus round.<br/>
+   * The round when this address book was created.
+   */
+  int64 round = 1;
+
+  /**
+   * The node ID of the next address that can be added.
+   * <p>
+   * The next node identifier assigned SHALL be equal to this value.<br/>
+   * All existing node identifiers SHALL be strictly less than this value.<br/>
+   */
+  NodeId next_node_id = 2;
+
+  /**
+   * A list of all consensus node addresses.
+   * <p>
+   * This list SHALL NOT be empty.
+   * If a consensus node is not in this list it SHALL NOT participate in the
+   * network.
+   */
+  repeated Address addresses = 3;
+}
+
+/**
+ * A single network address.
+ * This is one address in the network address book, including all required
+ * information to include that consensus node in the consensus gossip.
+ *
+ * ### Implementation Detail
+ * When `enableEventStreaming` is set to true, the memo field is REQUIRED
+ * and MUST be unique.
+ */
+message Address {
+  /**
+   * The ID of this member.
+   * <p>
+   * This identifier SHALL be agreed upon by all consensus nodes.
+   */
+  NodeId id = 1;
+
+  /**
+   * The nickname this consensus node uses to refer to another consensus node.
```

Comment:
Should this be "nickname other nodes use to refer to this node"?

Reply:
No, actually this nickname is a name that the current node uses to refer to the other consensus node that is in its `AddressBook`. 

That is, we have something like this for the node0

|| address || nickname ||
|0.0.4 | Swirlds |
|  0.0.5 | FIS |
| 0.0.6 | Wipro |

Reply:
This doesn't say what you're describing because the context is not "the node reading config.txt"  The context is "the node described in this individual message".

"This consensus node" in context only has this one nickname, so there is no way it can map it's own nickname to other nodes (unless each node refers to _every_ other node by it's own nickname).

If this is temporary we can ignore the incongruity of this text, but it's definitely not saying what you're describing.

Reply:
This is how javadoc looks like for the field
```
    /** name this member uses to refer to that member */
    private String nickname;
```

I'm not sure if I understand how you'd like to rephrase it, given the fact that this nickname is the information about other node that this node have.

Reply:
The perspective is reversed in the protobuf from the java, so this should work well:
```suggestion
   * The nickname other consensus nodes will use to refer to this consensus node.
```